### ISSUE TITLE
Generation of simple geometry meshes

### ISSUE THREAD

**Issue by wanghao-98** [CloudCompare/CloudCompare#2081]
When generating the simple geometry of box, its mesh contains 24 vertices, and the box itself only needs 8 vertices to represent, and 24 vertices cannot ensure the closure of the mesh, it is just a series of triangle stacking, which has a great impact on the subsequent processing, I hope these details can be improved, thank you!
![image](https://github.com/user-attachments/assets/26087d15-8016-4b03-9536-8315a5916111)
Boundary edge detection display, where green is the boundary edge

**Comment by dgirardeau** [comment]
Indeed, CC seems to be using 6 independent planes to create a box primitive:
https://github.com/CloudCompare/CloudCompare/blob/master/libs/qCC_db/src/ccBox.cpp#L39

**Comment by wanghao-98** [comment]
Thank you for your reply! I don't care about how CC generates box, I hope the end result will be as expected of the box mesh (it's a closed grid); The specific generation method should be uncomplicated, I hope these details can be optimized, thank you again for your contributions, thank you!

**Comment by dgirardeau** [comment]
This is now fixed: 245abf6bd60dddccf07e2e17773b49ab9f39adb9

A new alpha release will come out soon.
